apps/api/src/services/api.service.ts

```diff
@@ -60,26 +125,9 @@ export class ApiService implements OnApplicationShutdown {
   //   });
   // }
 
-  // async getGraphs(queryParams: GraphsQueryParamsDto): Promise<UserGraphDto[]> {
-  //   const { msaIds, privacyType } = queryParams;
-  //   const graphKeyPairs = queryParams.graphKeyPairs || [];
-  //   const graphs: UserGraphDto[] = [];
-  //   // eslint-disable-next-line no-restricted-syntax
-  //   for (const msaId of msaIds) {
-  //     const dsnpUserId: MessageSourceId = this.blockchainService.api.registry.createType('MessageSourceId', msaId);
-  //     // eslint-disable-next-line no-await-in-loop
-  //     const graphEdges = await this.asyncDebouncerService.getGraphForMsaId(dsnpUserId, privacyType, graphKeyPairs);
-  //     graphs.push({
-  //       msaId,
-  //       dsnpGraphEdges: graphEdges,
-  //     });
-  //   }
-  //   return graphs;
-  // }
-
   // eslint-disable-next-line class-methods-use-this
-  // private calculateJobId(jobWithoutId: ProviderGraphDto): string {
-  //   const stringVal = JSON.stringify(jobWithoutId);
-  //   return createHash('sha1').update(stringVal).digest('base64url');
-  // }
+  private calculateJobId(jobWithoutId): string {
```

Comment:
Not a blocker, but since we're using this in a couple of places, wondering if we want to make it reusable.